lib/gds_api/test_helpers/email_alert_api.rb

```diff
@@ -227,6 +226,7 @@ def stub_email_alert_api_does_not_have_subscriber_list_by_slug(slug:)
       alias_method :email_alert_api_has_subscriber_subscriptions, :stub_email_alert_api_has_subscriber_subscriptions
       alias_method :email_alert_api_does_not_have_subscriber_subscriptions, :stub_email_alert_api_does_not_have_subscriber_subscriptions
       alias_method :email_alert_api_has_subscription, :stub_email_alert_api_has_subscription
+      alias_method :email_alert_api_has_subscriptions, :stub_email_alert_api_has_subscriptions
```

Comment:
Was this missed in the previous PR?

Reply:
No, the previous PR added a new API method that can be used by email-alert-frontend.

This PR adds a new test helper, which needs exposing here.

Reply:
I think this is in the wrong commit then, as `stub_email_alert_api_has_subscriptions` doesn't yet.

Reply:
Whoops!